pitop/processing/algorithms/faces/face_detector.py

```diff
@@ -0,0 +1,114 @@
+import os
+import cv2
+import dlib
+from pitop.processing.utils.vision_functions import (
+    resize,
+    center_reposition
+)
+from .face_utils import get_face_angle
+from pitop.core import ImageFunctions
+from imutils import face_utils
+
+# directory where calibration output pickle file is located
+classifier_dir = 'predictors'
+script_dir = os.path.dirname(os.path.realpath(__file__))
+abs_file_path = os.path.join(script_dir, classifier_dir)
+
+# Filename for predictor
+predictor_file_name = "shape_predictor_68_face_landmarks.dat"
```

Comment:
This is ~100MB, but is necessary for the emotion/expression classification which will be done soon

Reply:
can we download it upon request? In case someone tried to do the challenge. Also, I'm wondering if we can compress it and decompress upon installation

Reply:
How is this file provided?